src/growthbook.ts

```diff
@@ -96,4 +97,8 @@ export const features = {
     maxPosts: 5,
     longTextLimit: 150,
   }),
+  personalizedDigestSendType: new Feature(
+    'personalized_digest_send_type',
+    UserPersonalizedDigestSendType.weekly,
+  ),
```

Comment:
Separated sendType to different feature flag since we went with Option 1 from RFC regarding allocations.

Reply:
Perfect for now